lib/fse.h

```diff
@@ -45,6 +45,32 @@ extern "C" {
 #include <stddef.h>    /* size_t, ptrdiff_t */
 
 
+/*-*****************************************
+*  FSE_PUBLIC_API : control library symbols visibility
+******************************************/
+#if defined(__GNUC__) && (__GNUC__ >= 4)
+#  define FSE_PUBLIC_API __attribute__ ((visibility ("default")))
```

Comment:
I renamed the option to `FSE_DLL_EXPORT`. That way they should only be marked as public when explicitly built with `FSE_DLL_EXPORT=1` (which is default for libfse, but not for libzstd)

As an added bonus, people who want to include FSE exports in libzstd as well, can just set it to 1, and use a single file.